### PR TITLE
I added a contributing.md file to the project

### PR DESCRIPTION
**Pull request by Charity10**
The code of conduct file is just a suggestion i haven't created the file for that yet. 
I'm open to corrections, addition or subtraction on the CONTRIBUTING.md file.
Thank you.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+4. **Documentation**: If you are improving documentation, please follow the existing style and make sure your documentation is clear and concise.
+
+## Code of Conduct
+
+By participating in this project, you agree to abide by our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Contact
+
+If you have questions, concerns, or need to contact the project team, you can reach out to [maintainer email or contact information].
```
Comment (kohasummons) [inline CONTRIBUTING.md:50]: We have a discord community. I think that would be a preferable point of contact. 

https://discord.com/invite/dUQyb2mk4s

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Getting Started
+
+Before you begin contributing, please follow these steps:
+
+1. Fork the repository to your GitHub account.
+2. Clone your forked repository to your local machine and add upstream remote:
+
+```bash
+git clone https://github.com/dxmentorship/dxmentorship.git
```
Comment (kohasummons) [inline CONTRIBUTING.md:24]: I think this should be 

```bash
git clone https://github.com/[YOUR-GITHUB-USERNAME]/dxmentorship.git
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+cd dxmentorship
+git remote add upstream https://github.com/dxmentorship/dxmentorship.git
```
Comment (kohasummons) [inline CONTRIBUTING.md:26]: The URL also would change to reflect the fork just like the one above.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+   - Test your changes and ensure that they do not introduce new issues.
+   - Submit a pull request to the main repository with a clear description of your changes.
+
+3. **Code Review**: We welcome code reviews from contributors. If you are not a developer but would like to help review code or documentation, please feel free to comment on pull requests.
+
+4. **Documentation**: If you are improving documentation, please follow the existing style and make sure your documentation is clear and concise.
+
+## Code of Conduct
+
+By participating in this project, you agree to abide by our [Code of Conduct](CODE_OF_CONDUCT.md).
```
Comment (kohasummons) [inline CONTRIBUTING.md:46]: We do not currently have a Code Of Conduct Document 😭. Would you be sending in a contribution to fix that?

**Review by kohasummons — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 4 above)
Comment (Charity10) [reply CONTRIBUTING.md:46]: Yes, I understand that there is currently no Code of Conduct document in the project. My mention of it was indeed a suggestion. I'll be happy to contribute to creating one. Please let me know if there are specific guidelines you'd like to include in the document.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Charity10) [reply CONTRIBUTING.md:26]: Thank you for the feedback 
I will make all necessary adjustments

File: CONTRIBUTING.md (hunk 4 above)
Comment (kohasummons) [reply CONTRIBUTING.md:46]: Awesome, thank you. Nope, there are no specific guidelines.

**Review by kohasummons — APPROVED** [review]
Looks Good!

**Pull request merged**